maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/booterclient/ForkStarter.java

```diff
@@ -594,9 +591,8 @@ private RunResult fork( Object testSet, KeyValueSource providerProperties, ForkC
             cli.createArg().setValue( systPropsFile.getName() );
         }
 
-        final ThreadedStreamConsumer threadedStreamConsumer = new ThreadedStreamConsumer( forkClient );
-        final CloseableCloser closer = new CloseableCloser( forkNumber, threadedStreamConsumer,
-                                                            requireNonNull( testProvidingInputStream, "null param" ) );
+        ThreadedStreamConsumer threadedStreamConsumer = new ThreadedStreamConsumer( forkClient );
```

Comment:
final?

Reply:
This part of code will be moved a bit and reworked due to both principles have different requirements (pipes use std/out as event receiver, but Netty does not).
Final was removed becuase line lenght was longer than 120 chars. I use final in these cases when having annonymous classes or Lambda but this is not the case. I guess having the statement in one line and not in two or three is more readable. WDYT?

I am fixing the unit tests in `maven-surefire-common` and then I will help you adapting this class. You will have more simple life because you will implement only new staff. I will try to make several commits with different concerns - easier for making review. Then we will squash the commits after we would be happy with our work.